src/app/styles/mixins/mixins.scss

```diff
@@ -25,12 +24,37 @@
   flex-direction: row;
   align-items: flex-start;
   justify-content: flex-start;
-
 }
 
 @mixin flexRowEnd {
   display: flex;
   flex-direction: row;
   align-items: flex-end;
   justify-content: flex-end;
-}
\ No newline at end of file
+}
+
+@mixin flexRowCentered {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+@mixin shadowEffect {
+  box-shadow: 5px 5px 10px $color_shadow;
+  border: {
```

Comment:
change to: border: 1px solid $color_background-light-ivory;

Reply:
changed